book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,235 @@
+  involved with a PR regarding a tier 3 platform, unless they have opted into
+  such messages.
+- Patches adding or updating tier 3 platforms must not break any existing tier 2
+  or tier 1 platform, and must not knowingly break another tier 3 platform
+  without approval of either the Zebra team of the other tier 3 platform.
+
+If a tier 3 platform stops meeting these requirements, or the platform maintainers
+no longer have interest or time, or the platform shows no signs of activity and
+has not built for some time, or removing the platform would improve the quality
+of the Zebra codebase, we may post a PR to remove support for that platform. Any such PR will be CCed
+to the platform maintainers (and potentially other people who have previously
+worked on the platform), to check potential interest in improving the situation.
+
+## Tier 2 platform policy
+
+At this tier, the Zebra project guarantees that a platform builds, and will reject
+patches that fail to build on a platform. Thus, we place requirements that ensure
+the platform will not block forward progress of the Zebra project.
+
+A proposed new tier 2 platform must be reviewed and approved by Zebra team based
+on these requirements.
+
+In addition, the devops team must approve the integration of the platform
+into Continuous Integration (CI), and the tier 2 CI-related requirements. This
+review and approval may take place in a PR adding the platform to CI, or simply
+by a devops team member reporting the outcome of a team discussion.
+
+- Tier 2 platforms must implement all the Zcash consensus rules.
+  Other Zebra features and binaries may be disabled, on a case-by-case basis.
+- A tier 2 platform must have a designated team of developers (the "platform
+  maintainers") available to consult on platform-specific build-breaking issues.
+  This team must have at least 1 developer.
+- The platform must not place undue burden on Zebra developers not specifically
+  concerned with that platform. Zebra developers are expected to not gratuitously
+  break a tier 2 platform, but are not expected to become experts in every tier 2
+  platform, and are not expected to provide platform-specific implementations for
+  every tier 2 platform.
+- The platform must provide documentation for the Zcash community explaining how
+  to build for their platform, and explaining how to run tests for the platform.
+  If at all possible, this documentation should show how to run Zebra programs
+  and tests for the platform using emulation, to allow anyone to do so. If the
+  platform cannot be feasibly emulated, the documentation should document the
+  required physical hardware or cloud systems.
+- The platform must document its baseline expectations for the features or
+  versions of CPUs, operating systems, and any other dependencies.
+- The platform must build reliably in CI, for all components that Zebra's CI
+  considers mandatory.
```

Comment:
```suggestion
- The platform must build reliably in CI, for all components that Zebra's CI
  considers mandatory.
  - Since a working Rust compiler is required to build Zebra,
    the platform must be a [Rust tier 1 platform](https://rust-lang.github.io/rustup-components-history/).
```